flink-runtime/src/main/java/org/apache/flink/runtime/instance/SettableSlotContext.java

```diff
@@ -26,9 +26,9 @@
 import org.apache.flink.util.Preconditions;
 
 /**
- * Simple implementation of the {@link SlotContext} interface for the legacy code.
+ * A settable implementation of the {@link SlotContext} interface.
  */
-public class SimpleSlotContext implements SlotContext {
+public class SettableSlotContext implements SlotContext {
```

Comment:
why is it renamed to `SettableSlotContext`? In particular, what is meant by `settable`?

Reply:
It follows the convention implied by `SettableLeaderRetrievalService`. In fact, I'd prefer move it to "test scope", whether rename or not is not quite important for me.

Reply:
I would also lean towards having it in test package and name it `TestingSlotContext` but it is still part of legacy code. `SettableLeaderRetrievalService` looks like something you can manually set or notify. `SimpleSlotContext` is just a simple wrapper of final variables, what is to be set there? I would leave the name like it is now and rename it to `TestingSlotContext` when it is used only in tests and it is possible to move to it to test package.